tests/API_smoke_test.postman_collection.json

```diff
@@ -464,6 +465,3139 @@
+										}
+									]
+								},
+								"method": "POST",
+								"header": [],
+								"url": {
+									"raw": "{{api}}/auth/login",
+									"host": [
+										"{{api}}"
+									],
+									"path": [
+										"auth",
+										"login"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Create custodial profile",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "9ca1ca55-a3d4-4146-8b96-5ce198579252",
+										"exec": [
+											"const assert = eval(globals.loadAsserts);",
+											"assert.common(200, 500, false, pm.response.json(),\"userProfileSchema\");"
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "prerequest",
+									"script": {
+										"id": "41dbe3a9-efca-4b72-a543-b58ac98dc070",
+										"exec": [
+											"var moment = require('moment');\r",
+											"let newMoment = moment().subtract(18, 'years');\r",
+											"pm.variables.set(\"birthdate\", newMoment.format('YYYY-MM-DD'));"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "PUT",
+								"header": [
+									{
+										"key": "x-tidepool-session-token",
+										"type": "text",
+										"value": "{{sessionToken}}"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\"fullName\":\"{{$randomFullName}}\",\"patient\":{\"birthday\":\"{{birthdate}}\",\"mrn\":\"{{$randomInt}}\",\"targetTimezone\":\"US/Eastern\"}}",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{api}}/metadata/{{custodialUserId}}/profile",
+									"host": [
+										"{{api}}"
+									],
+									"path": [
+										"metadata",
+										"{{custodialUserId}}",
+										"profile"
+									]
+								},
+								"description": "Setting up Name, clinic name, role and telephone number."
+							},
+							"response": []
+						},
+						{
+							"name": "Upload device data",
+							"event": [
+								{
+									"listen": "prerequest",
+									"script": {
+										"id": "8b9cf43a-8c79-47b0-9e17-02f9fb0214ba",
+										"exec": [
+											"const uuid = require('uuid');",
+											"pm.variables.set(\"uploadDeviceDataItemId\", uuid.v4());",
+											"const moment = require('moment');",
+											"let now = moment();",
+											"pm.variables.set(\"uploadDeviceDataItemTime\", now.format());",
+											"pm.variables.set(\"uploadDeviceDataItemDeviceTime\", now.format('YYYY-MM-DDThh:mm:ss'));",
+											"",
+											"function randomString(length, chars) {",
+											"    var result = '';",
+											"    for (var i = length; i > 0; --i) result += chars[Math.round(Math.random() * (chars.length - 1))];",
+											"    return result;",
+											"}",
+											"",
```

Comment:
I think you can shorten this by using similar / the same to here https://github.com/tidepool-org/APITests/blob/d22349a8a40ce3cce194f47edcdd0a45310b1dca/tests/API_smoke_test.postman_collection.json#L271

Reply:
good suggestion. replaced with `pm.variables.set("uploadId", uuid.v4().replace(/-/g, '').slice(0,12));`  in order to simulate a closer version of the upload id we create